### DISCUSSION TITLE
cmd-q and cmd-w forcefully exiting you from the song mid play

### DISCUSSION THREAD

**Discussion by zhengno** [ppy/osu#30050] [Ideas]
I play with QWOP. This is very frustrating, since it's usually in the middle of a song. Please add a keybind option for both of them so I can clear them permanently

https://github.com/user-attachments/assets/587be558-cd15-4df4-9e61-23534803fb4f

**Comment by peppy** [comment]
On macOS? These are *system level binds*. If you really want to disable them, you'll need to use something like Karabiner Elements or skhd.
